sdk/tables/azure-data-tables/CHANGELOG.md

```diff
@@ -1,14 +1,14 @@
 # Release History
 
-## 12.4.0-beta.1 (Unreleased)
+## 12.4.0-beta.1 (2024-03-13)
 
-### Features Added
+### Other Changes
 
-### Breaking Changes
+- Migrating from Jackson to stream-style serialization.
```

Comment:
May want a small comment that this removes `jackson-dataformat-xml` as a dependency which could be a breaking change if it was being depended on as a transitive dependency.